runtime/oti/j9port_generated.h

```diff
@@ -388,11 +388,16 @@ typedef struct J9PortLibrary {
 	/** see @ref j9portcontrol.c::j9port_control "j9port_control"*/
 	int32_t (*port_control)(struct J9PortLibrary *portLibrary, const char *key, uintptr_t value) ;
 #if defined(J9VM_OPT_CRIU_SUPPORT)
-	/* The delta between Checkpoint and Restore of j9time_nano_time() return values.
+	/* The delta between Checkpoint and Restore of j9time_current_time_nanos() return values.
 	 * It is initialized to 0 before Checkpoint, and set after restore.
 	 * Only supports one Checkpoint, could be restored multiple times.
 	 */
 	int64_t checkpointRestoreTimeDelta;
+	/* The delta between Checkpoint and Restore of j9time_current_time_nanos() return values.
+	 * It is initialized to 0 before Checkpoint, and set after restore.
+	 * Only supports one Checkpoint, could be restored multiple times.
+	 */
+	int64_t prePostCheckpointNanotimeClockDelta;
```

Comment:
i dont think we need both here, we only need the one that `j9time_nano_time` will use.

Reply:
`j9time_nano_time` uses the diff for automatic adjustment, the diff could be negative. The other one represents the JVM downtime
https://github.com/eclipse-openj9/openj9/blob/7b717f70683c05a5304f8861e2ee675380753e65/jcl/src/java.base/share/classes/openj9/internal/criu/InternalCRIUSupport.java#L39-L45

Reply:
yes, you can add a new field to `javaVM->checkpointState`

Reply:
Moved it to `J9JavaVM->checkpointState.checkpointRestoreTimeDelta`.

Reply:
you can rename to `nanoTimeMononicClockDelta`

Reply:
Renamed `prePostCheckpointNanotimeClockDelta` to `nanoTimeMononicClockDelta`.